lib/MetaCPAN/Document/Release/Set.pm

```diff
@@ -9,6 +9,34 @@ use MetaCPAN::Util qw( single_valued_arrayref_to_scalar );
 
 extends 'ElasticSearchX::Model::Document::Set';
 
+sub author_status {
+    my ( $self, $id, $file ) = @_;
+    return unless $id and $file;
+
+    my $st = $file->{_source}
```

Comment:
It's not clear to me what `$st` is.  Is this `status`?  If so, could we use that name?  :)

Reply:
I see this is a cut/paste, but might be a good time to clean up that variable name.

Reply:
sure. I'll do some cleaning up